src/Symfony/Component/Validator/Constraints/Email.php

```diff
@@ -74,5 +75,9 @@ public function __construct($options = null)
             // throw new LogicException(sprintf('The "egulias/email-validator" component is required to use the "%s" constraint.', __CLASS__));
             @trigger_error(sprintf('Using the "%s" constraint without the "egulias/email-validator" component installed is deprecated since Symfony 4.2.', __CLASS__), E_USER_DEPRECATED);
         }
+
+        if (null !== $this->normalizer && !\is_callable($options['normalizer'])) {
```

Comment:
I think we could be consistent here in accessing either the property or the array element. Could be less confusing.

Reply:
Good catch! Thanks.